homepage/script/jsonresp.js

```diff
@@ -0,0 +1,100 @@
+var urls = ["https://fsd1.herokuapp.com/users/1/details",
+"https://fsd1.herokuapp.com/users/1/followers/suggestions",
+"https://fsd1.herokuapp.com/users/1/following/tweets"];
+
+var consolidate = urls.map(url => fetch(url).then(res => res.json()));
+Promise.all(consolidate).then(res => acceptURLs.call(res));
+
+var acceptURLs = function() {
+	updateDOM.apply(this[0]);
+	updateSuggestions.apply(this[1]);
+	tweetContent.apply(this[2]);
+}
+
+var updateDOM = function() {
```

Comment:
This is just the profile info and not the whole DOM manipulation. Use more specific function names